sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -38,6 +38,7 @@ internal class ViewAnnotationManagerImpl(
   internal val idLookupMap = ConcurrentHashMap<View, String>()
   private val currentlyDrawnViewIdSet = mutableSetOf<String>()
   private val unpositionedViews = mutableSetOf<View>()
+  private val annotationsMap = ConcurrentHashMap<View, ViewAnnotationOptions>()
```

Comment:
maybe I missed smth but when do you add elements to with map?

Reply:
it's happening on L312 with `map[key]=value` syntax